Aerospike_Nozzle/SPEXpdd.tex

```diff
@@ -0,0 +1,292 @@
+	% Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+	% What matters are what questions you intend to answer.
+	% This is the main purpose or main goal the project hopes to achieve.
+	
+	The primary objective of this project is to further characterize flow of aerospike nozzles by building upon current simulation methods and models through mathematical analysis and physical validation (i.e. cold flow experiments). This experience and knowledge will concurrently be applied to the design on an aerospike nozzle to propel a sounding rocket to 100 km. 
+	
+	As of this document's latest revision, there are no secondary objectives.
+	
+	
+	% \section{Secondary Objectives}
+	% \label{sec:secondary-obj}
+	% Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+	
+	\section{Benefit to SPEX}
+	\label{sec:benefit}
+	% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+	% and to challenge them with interesting projects.
+	% In this section, explain how the project would benefit SPEX members as students,
+	% space enthusiasts, and young professionals.
+	
+	Members of SPEX who are involved in this project will acquire a host of technical skills related to simulation, modeling, programming, design engineering, data acquisition, and experimental analysis. These skills will play a critical role in future SPEX Propulsion projects as well as SPEX-wide endeavors as well, such as the NASA CubeSat Launch Initiative. Employers are in-search of those with experience in analysis, physical validation, and how the iterative relationship between them can yield a better end-product.
+	
+	Additionally, this project's findings will be thoroughly documented, culminating in a paper, presentation, and poster to exhibit at events such as the Undergraduate Research Symposium, ImagineRIT, and an AIAA technical conference. Attending these events and presenting our published work will help establish SPEX in the science and aerospace communities.
+	
+	\section{Implementation}
+	\label{sec:implementation}
+	
+	This project is expected to take two semesters. The first semester will be devoted to developing the MATLAB script and running simulations. Initial cold flow test results from \textit{Investigation of a Cold Gas Thruster} will help validate values outputted by scripts and simulations, as related to CBNs. From there, CFD will be used to validate the MATLAB script when aerospike nozzle contouring is undertaken. The first semestester will also include the mechanical design of the aerospike nozzle and the low pressure chamber used to simulate ascent through the atmosphere. During this physical simulation, we will be able to observe how the exhaust plume changes with respect to ambient pressure and thrust data points.
+	
```

Comment:
What measurements do you expect to take for these tests? (Temperature, pressure? How many sensors? Steady state? Schlieren lensing, thermal imaging or other optical techniques?) Bonus: why?